src/glow/util/RoSeCamera.cpp

```diff
@@ -237,11 +237,72 @@ bool RoSeCamera::mouseMoved(float x, float y, MouseButton btn, KeyboardModifier
 }
 
 bool RoSeCamera::wheelEvent(float delta, KeyboardModifier modifier) {
+  mutex_.lock();
+  static const float ZOOM_SENSITIVITY = 3.f;
   // move along the viewing direction specified by yaw and pitch.
+  
+    
+    float forward = ZOOM_SENSITIVITY * delta * std::cos(pitch_);    
+    float up = ZOOM_SENSITIVITY * delta * std::sin(pitch_) * (-1);
+    float s = std::sin(yaw_);
+    float c = std::cos(yaw_);
+
+
 
+    x_ -= forward * s;
+    y_ -= up;
+    z_ += forward * c * (-1);
   // TODO: implement me!
+   mutex_.unlock();
 
   return true;
 }
 
+bool RoSeCamera::keyPressed(KeyboardKey key, KeyboardModifier modifier){  
+  switch(static_cast<std::underlying_type<KeyboardKey>::type>(key)){
+    case 65: //a
```

Comment:
Not sure, but I would prefer the more verbose `KeyboardKey::KeyA` even though this needs the `if` statements.